### ISSUE TITLE
Feature Request: Map chrome's own shortcuts

### ISSUE THREAD

**Issue by IfMung** [philc/vimium#2980]
Although vimium can map a lot of shortcuts, but still lack a lot of shortcuts. So I was wondering if could add the ability of vimium to map chrome's own shortcuts.

**Comment by smblott-github** [comment]
Do you mean something like:

    map X chome key=<c-l>

to focus the address bar?

I don't think that there's an API for that.

**Comment by IfMung** [comment]
Yes, forgive me for my poor English and thank for your reply : )

I offent  focus on chrome address bar or other situation that lost foucs on page, And have to use the mouse to switch focus on the page so that I can continue to use vimium.  It's so boring.

And is any solution about it?

**Comment by smblott-github** [comment]
Not really.  There's no API for that.  I think maybe `<Tab>` sometimes puts the focus back in the page, but I usually just tap.

**Comment by innaterebel** [comment]
Although not located at a super convenient place, <kbd>F6</kbd> is Chrome's native key to cycle focus between omnibar＞bookmark bar (when visible)＞page content＞download shelf (when visible).
